### PR TITLE
Change hiera.yaml location from $codedir to $confdir

### PR DESCRIPTION
**Pull request by greglarkin**

### CODE REVIEW

File: courses/published/getting_started_with_hiera.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ file { '/etc/motd':
...
 You'll need to tell Hiera where to find the data, you can do this by creating a
-file called `/etc/puppetlabs/code/hiera.yaml`:
+file called `/etc/puppetlabs/puppet/hiera.yaml`:
 
 <pre>
 ---
 :backends: "yaml"
 :yaml:
-  :datadir: "/etc/puppetlabs/code/hieradata/"
+  :datadir: "/etc/puppetlabs/puppet/hieradata/"
```
Comment (samuelson) [inline courses/published/getting_started_with_hiera.md]: @greglarkin this path should still be code

Comment (greglarkin) [inline courses/published/getting_started_with_hiera.md]: @samuelson Oh yeah, that's true. Do you want to keep hieradata there to keep things simple or go for the /etc/puppetlabs/code/environments/production/hieradata route that will be encountered in a default PE install?

Comment (samuelson) [inline courses/published/getting_started_with_hiera.md]: Let's keep it in this dir.  I want to make it simple and adding in environments will get confusing quickly.

Comment (greglarkin) [inline courses/published/getting_started_with_hiera.md]: Makes sense, I'll change it.

**Pull request merged**